web/pages/_app.tsx

```diff
@@ -4,7 +4,18 @@ import Head from 'next/head'
 import Script from 'next/script'
 import { usePreserveScroll } from 'web/hooks/use-preserve-scroll'
 
+function printBuildInfo() {
+  // undefined if e.g. dev server
+  if (process.env.NEXT_PUBLIC_VERCEL_ENV) {
```

Comment:
Would it be better to put this in a useEffect(() => { ... } ,[]) so that it only runs once?

Reply:
I put it here partly because this seemed like the most obvious place where it would run exactly once per page -- do you suspect it could run more than once? If so, sure, I will add that.